modules/mobile/app/models/mobile/v0/adapters/lighthouse_claims_overview.rb

```diff
@@ -9,7 +9,7 @@ class LighthouseClaimsOverview < ClaimsOverview
         def parse(list)
           list
             .map { |entry| entry['type'] == 'claim' ? parse_claim(entry) : parse_appeal(entry) }
-            .sort_by(&:updated_at).reverse!
+            .sort_by { |entry| [entry[:updated_at] ? 1 : 0, entry[:updated_at]] }.reverse!
```

Comment:
This is to ensure that all nils end up at the end, right?

Reply:
yup

Reply:
I thought `sort_by` already treats nil as 0?

Reply:
updated_at is populated by the upstream field `attributes['claimPhaseDates']['phaseChangeDate']` rather than a sql record save.

Reply:
After looking closer at the code, I see what you're saying. This error could be not happening because it fails just before this.